### ISSUE TITLE
Change package name to avoid conflicting names

### ISSUE THREAD

**Issue by c0nsol3** [Solar-network/python-crypto#21]
Same as https://github.com/ArkEcosystem/python-crypto/issues/124

TLDR: `crypto` is too generic name for a library. There are other libraries (eg Ark's) which uses the same name. This means you can't have conflicting libraries installed at the same time.

I propose we change the name of this library to `solar_crypto`. Please note that only the library name (when importing it) would be changed.

#### Curent
```py
from crypto.identity import address
```

#### Proposed
```py
from solar_crypto.identity import address
```

**Comment by c0nsol3** [comment]
I'll make a PR for this ready for review after currently open PRs get merged. After we're satisfied with the name and decide to merge it I'll publish the python package to pypi.

**Comment by alessiodf** [comment]
Any reason we can't use kebab case instead of snake case @c0nsol3?

**Comment by c0nsol3** [comment]
Python doesn't support importing names with a dash, eg:

`from solar-crypto.identity import address` will get you a syntax error `SyntaxError: invalid syntax`

To clarify my name suggestion, this is how it would be used in practice:

- When installing: `pip install solar-crypto`
- When importing: `from solar_crypto.identity import address`

This is a pretty standard naming convention in Pythonland.

**Comment by sleepdefic1t** [comment]
As it appears this _does_ follow proper naming conventions, I'm good with it 👍

> Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. Python packages should also have short, all-lowercase names, although the use of underscores is discouraged.
> [PEP 8 – Style Guide: Package and Module Names](https://peps.python.org/pep-0008/#package-and-module-names)

**Comment by sleepdefic1t** [comment]
Resolved in #28
